src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -26,14 +37,16 @@ const CrossPodWriteForm = () => {
   const { session } = useSession();
   const { state, dispatch } = useStatusNotification();
   const { selectedUser } = useContext(SelectUserContext);
-  const [username, setUsername] = useState('');
+  const [username, setUsername] = useState(selectedUser.username);
```

Comment:
The initial state for username should be an empty string `''`. The state selectedUser is only triggered if selectedUser exist, which would not always be the case.